website/docs/r/mssql_database.html.markdown

```diff
@@ -0,0 +1,110 @@
+---
+subcategory: "Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_mssql_database"
+description: |-
+  Manages a MS SQL Database.
+---
+
+# azurerm_mssql_database
+
+Manages a MS SQL Database.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_sql_server" "example" {
+  name                         = "example-sqlserver"
+  resource_group_name          = azurerm_resource_group.example.name
+  location                     = azurerm_resource_group.example.location
+  version                      = "12.0"
+  administrator_login          = "4dm1n157r470r"
+  administrator_login_password = "4-v3ry-53cr37-p455w0rd"
+}
+resource "azurerm_mssql_database" "test" {
+  name           = "acctest-db-%d"
+  sql_server_id  = azurerm_sql_server.test.id
+  collation      = "SQL_Latin1_General_CP1_CI_AS"
+  license_type   = "LicenseIncluded"
+  max_size_gb    = 4
+  read_scale     = "Enabled"
+  sku_name       = "BC_Gen5_2"
+  zone_redundant = true
+
+  tags = {
+    foo = "bar"
+  }
+
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Ms SQL Database. Changing this forces a new resource to be created.
+
+* `sql_server_id` - (Required) The id of the Ms SQL Server on which to create the database. Changing this forces a new resource to be created.
```

Comment:
As mentioned above, possibly should be renamed to avoid confusion in future?  Also, possibly add a note to avoid confusion for "Serverless" still needing a Server resource to be associated to?
```suggestion
* `sql_server_id` - (Required) The id of the Ms SQL Server on which to create the database. Changing this forces a new resource to be created.
~> **NOTE:** This setting is still required for "Serverless" SKU's
```